### PR TITLE
Initial commit 

### PR DESCRIPTION
**Pull request by rado-straka**
New base image for https://portal.digitalsafe.net/browse/SCMOD-7373
http://sou-jenkins2.hpeswlab.net/job/CAFapi/job/CAFapi%7Eopensuse-jepfasttext-image%7ESCMOD-7373%7ECI/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# openSUSE and JepFastText image
+
+This project builds on the openSUSE JDK8 image [here](https://github.com/CAFapi/opensuse-java8-images/blob/develop/src/main/docker/jdk/Dockerfile), Python 3, Jep and fastText are then installed on the image.
+Jep is a library used for executing python scripts from within java code, more information on Jep can be found [here](https://github.com/ninia/jep/blob/master/README.rst).
```
Comment (michael-bryson) [inline README.md]: If this is supposed to be a newline it does not display like that in GitHub so you will need to change the format if that is what you are looking for.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+FastText is an open-source, free, lightweight library that allows users to learn text representations and text classifiers for language detection, more information on fastText can be found [here](https://github.com/facebookresearch/fastText/).
```
Comment (michael-bryson) [inline README.md]: Same as above.

**Review by michael-bryson — COMMENTED** [review]
Can you also create a dev build for this to ensure it builds correctly on Jenkins before merging?

**Comment by rado-straka** [comment]
Updated and added forgotten link to build

**Review by andyreidz — APPROVED** [review]

**Review by michael-bryson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andyreidz) [reply README.md]: It's ok the bit in [] is the display text for the hyperlink and the () is the url

**Pull request merged**